chain/txValidator.go

```diff
@@ -165,15 +165,37 @@ func CheckTransactionPayload(txn *transaction.Transaction) error {
 		if Fixed64(pld.RegistrationFee) < Fixed64(config.MinGenIDRegistrationFee) {
 			return errors.New("fee is too low than MinGenIDRegistrationFee")
 		}
+	case pb.NanoPayType:
+		pld := payload.(*pb.NanoPay)
 
+		if len(pld.Sender) != 20 && len(pld.Recipient) != 20 {
```

Comment:
It's better to use UINT160SIZE instead of hard coded 20

Reply:
@NoelBright We should also modify the existing ones

Reply:
modified wherever found